README.md

```diff
@@ -54,64 +55,96 @@ var minioClient = new Minio.Client({
 
 ## Quick Start Example - File Uploader
 
-This example program connects to an object storage server, makes a bucket on the server and then uploads a file to the bucket.
+This sample ECMAScript module connects to an object storage server, creates a bucket, and uploads a file to the bucket.
+It uses the MinIO `play` server, a public MinIO cluster located at [https://play.min.io](https://play.min.io).
 
-We will use the MinIO server running at [https://play.min.io](https://play.min.io) in this example. Feel free to use this service for testing and development. Access credentials shown in this example are open to the public.
+The `play` server runs the latest stable version of MinIO and may be used for testing and development.
+The access credentials shown in this example are open to the public.
+All data uploaded to `play` should be considered public and non-protected.
 
-#### file-uploader.js
+#### file-uploader.mjs
 
 ```js
-var Minio = require('minio')
+import * as Minio from 'minio'
 
-// Instantiate the minio client with the endpoint
-// and access keys as shown below.
-var minioClient = new Minio.Client({
+// Instantiate the MinIO client with the endpoint,
+// access, and secret keys
+// play.min.io is the MinIO public test cluster
+const minioClient = new Minio.Client({
   endPoint: 'play.min.io',
   port: 9000,
   useSSL: true,
   accessKey: 'Q3AM3UQ867SPQQA43P2F',
   secretKey: 'zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG',
 })
 
-// File that needs to be uploaded.
-var file = '/tmp/photos-europe.tar'
+// File to upload
+const sourceFile = '/tmp/test-file.txt'
 
-// Make a bucket called europetrip.
-minioClient.makeBucket('europetrip', 'us-east-1', function (err) {
-  if (err) return console.log(err)
+// Destination bucket
+const bucket = 'js-test-bucket'
 
-  console.log('Bucket created successfully in "us-east-1".')
+// Destination object name
+const destinationObject = 'my-test-file.txt'
 
-  var metaData = {
-    'Content-Type': 'application/octet-stream',
-    'X-Amz-Meta-Testing': 1234,
-    example: 5678,
+
+/* How to do this? exists is always undefined
+// Create the bucket if it doesn't already exist
+minioClient.bucketExists(bucket, function (err, exists) {
+  if (err) {
+    console.log(err)
+    return
```

Comment:
I was able to do this when it was still a `.js` file, but now I'm not sure how to get a value for `exists`. I've tried a couple different things but it's always undefined by the time I want to use it below. So I'm missing something about scope, apparently.

Reply:
rename it to mjs or create a async main function

Reply:
Aha! I tried several variations, but not removing the anonymous function entirely. That did it.